src/staticanalysis/bot/static_analysis_bot/config.py

```diff
@@ -69,20 +56,16 @@ def __init__(self):
 
     def setup(self,
               app_channel,
-              work_dir,
               publication,
               allowed_paths,
               cov_config=None,
-              max_clone_runtime=15*60,
-              build_plan=None
               ):
         # Detect source from env
         if 'TRY_TASK_ID' in os.environ and 'TRY_TASK_GROUP_ID' in os.environ:
-            self.source = SOURCE_TRY
             self.try_task_id = os.environ['TRY_TASK_ID']
             self.try_group_id = os.environ['TRY_TASK_GROUP_ID']
         else:
-            self.source = SOURCE_PHABRICATOR
+            raise Exception('Only TRY mode is supported')
 
         self.app_channel = app_channel
         self.download({
```

Comment:
I guess we don't need to download the clang-tidy configuration anymore

Reply:
We need it to get the publishable checks (still needed for `ClangTidyIssue.validates()`)

Reply:
We could move this stuff to the try task

Reply:
(not now, in the future)

Reply:
To do that, we would need :
1. `./mach static-analysis check` to add a 'publishable' value on its output, using the same checks list
2. Add 3rd party checks on mozlint & static-analysis check